.github/workflows/python_tools.yml

```diff
@@ -36,7 +36,6 @@ jobs:
         python setup.py install
       working-directory: ./tools/validators/ontology_validator
     - name: Ontology Yaml Validator Generator Tests 
-      if: (matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest') 
```

Comment:
can you please delete this line. It will run all of them if you dont have an if here